### ISSUE TITLE
Drake Visualize a KUKA arm ERROR: Error reading 'v' at line 3

### ISSUE THREAD

**Issue by dvogureckiy99** [RobotLocomotion/drake#16979]
I Build drake from source on ubuntu 20.04.
Source version: vtk version 9.1.0

When I try run this example [Visualize a KUKA arm](https://drake.guzhaoyuan.com/to-get-started)
I have trouble with this command: `./bazel-bin/tools/drake_visualizer`
```
ERROR: In /vtk/src/IO/Import/vtkOBJImporter.cxx, line 495
vtkOBJPolyDataProcessor (0x55b5aa616a20): Error reading 'v' at line 3
```

**Comment by dvogureckiy99** [comment]
I resolved this error on my Ubuntu 20.04 system by changing the locale settings.

    1. Run in terminal:
    sudo update-locale LC_NUMERIC=C
    2. Log out and then log in.

**Comment by jwnimmer-tri** [comment]
Yes, unfortunately `drake-visualizer` does not correctly ignore the user's locale when parsing `*.obj` files.  Per #16215 though, we don't intent to try to fix that.  The work-around posted above is the correct one.  (Setting the locale just for the visualizer binary is also fine.)
